src/status_im/chat/subs.cljs

```diff
@@ -53,6 +53,22 @@
   (fn [[chats id] [_ k chat-id]]
     (get-in chats [(or chat-id id) k])))
 
+(defn vec->map
```

Comment:
Can we give this a more descriptive name? Something about :name maybe

Reply:
also k isn't used

Reply:
yeah, it should be `k` instead of `:name`. More descriptive name like..? `vec->map-by-key` ? 
`commands-vector->commands-map`, `commands-vector->commands-map-by-name`? idk, all these variants look ugly anyway imo

Reply:
`vec->map-by-key` seems fine to me, at least more descriptive. Assuming we need it in more places, could also  inline it/hardcode :name if that's all we use it for (not in util ns)

Reply:
What's the reason `(set all-commands)` doesn't work btw?